Source/Charts/Renderers/BarChartRenderer.swift

```diff
@@ -310,14 +310,36 @@ open class BarChartRenderer: BarLineScatterCandleBubbleRenderer
                 // Set the color for the currently drawn value. If the index is out of bounds, reuse colors.
                 context.setFillColor(dataSet.color(atIndex: j).cgColor)
             }
-            
-            context.fill(barRect)
-            
-            if drawBorder
+
+            if dataProvider.isDrawRoundedBarEnabled
             {
-                context.setStrokeColor(borderColor.cgColor)
-                context.setLineWidth(borderWidth)
-                context.stroke(barRect)
+                let cornerRadius = CGSize(width: barRect.width / 2.0, height: barRect.width / 2.0)
+                #if os(OSX)
```

Comment:
Just FYI, I came across API like `CGPathCreateWithRoundedRect()` and `CGPathAddRoundedRect()`, looks like we can replace the UIBezierPath APIs?